src/main/java/edu/wpi/first/shuffleboard/sources/SourceType.java

```diff
@@ -9,7 +9,7 @@
 public enum SourceType {
 
   NONE(false, "", null),
-  STATIC(false, "example://", s -> DummySource.forTypes(DataType.forName(s)).get()),
+  STATIC(false, "example://", s -> DummySource.forTypes(DataType.forName(s)).orElse(null)),
```

Comment:
It should, `DataSource.none()` is explicitly intended to be used for "null" source values